src/admin/containers/letters.js

```diff
@@ -1,48 +1,72 @@
 import React from 'react'
 import { connect } from 'react-redux'
+import { isEmpty } from 'ramda'
 
 import { send_newsletter_post
        , send_sub_reminder_post
+       , send_subscription_due_post
+       , subscription_due_post_tab
        , toggle_recipient_list
        , show_letter
+       , SEND_NEWSLETTER_POST
+       , SEND_SUB_REMINDER_POST
+       , SUBSCRIPTION_DUE_POST_TAB
        } from '../redux/modules/letters/letters.js'
 
 import PostMembersSection from '../dumb_components/letters/post_members_section.js'
 import SubLettersSection from '../dumb_components/letters/sub_letters_section.js'
+import SubDueSection from '../dumb_components/sub_due_section.js'
 
-const Letters = ({ send_newsletter_post, send_sub_reminder_post, active_tab, ...other }) => {
+const Letters = ({ send_newsletter_post, send_sub_reminder_post, active_tab, subscription_due_post_tab, ...other }) => {
   return (
     <div className='top-letter-container'>
 
       <button
-        className={'letters-tab' + (active_tab === 'members' ? ' letters-tab-active' : '')}
+        className={'letters-tab' + (active_tab === SEND_NEWSLETTER_POST ? ' letters-tab-active' : '')}
         onClick={send_newsletter_post}>
         Post Members
       </button>
 
       <button
-        className={'letters-tab' + (active_tab === 'letters' ? ' letters-tab-active' : '')}
+        className={'letters-tab' + (active_tab === SEND_SUB_REMINDER_POST ? ' letters-tab-active' : '')}
         onClick={send_sub_reminder_post}>
         Subscription Reminders
       </button>
 
+      <button
+        className={'letters-tab' + (active_tab === SUBSCRIPTION_DUE_POST_TAB ? ' letters-tab-active' : '')}
+        onClick={subscription_due_post_tab}>
+        Subscriptions Due
+      </button>
+
       <div>
-        {active_tab &&
-          (active_tab === 'letters'
-           ? <SubLettersSection {...other} />
-           : <PostMembersSection {...other} />)
-          }
+        {active_tab && map_tab[active_tab]({...other})}
       </div>
-
     </div>
   )
 }
 
-const mapStateToProps = (state) => (
-  { letters: state.letters
-  , active_tab: state.letters.active_tab
+const map_tab =
+  { [SEND_NEWSLETTER_POST]: props => <PostMembersSection {...props} />
```

Comment:
Same as above really. Also, if you define these as function components, most of these could just be changed to read more like

```js
[SEND_NEWSLETTER_POST]: PostMembersSection
```